controllers/component_build_controller.go

```diff
@@ -672,9 +667,19 @@ func GeneratePipelineRun(component appstudiov1alpha1.Component, bundle string, o
 			Params: params,
 			Workspaces: []tektonapi.WorkspaceBinding{
 				{
-					Name:                  "workspace",
-					PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{ClaimName: "appstudio"},
-					SubPath:               pipelineName + "-{{revision}}",
+					Name: "workspace",
+					VolumeClaimTemplate: &corev1.PersistentVolumeClaim{
```

Comment:
Does it makes sense to have a function that generates the `VolumeClaimTemplate`? Then use the function in application-service and here, so we have consistency on how the volume is created.

Reply:
Good point :+1: